panel/src/components/Forms/Blocks/BlockPasteboard.vue

```diff
@@ -1,11 +1,11 @@
 <template>
 	<k-dialog
 		ref="dialog"
+		:class="['k-block-importer', $attrs.class]"
```

Comment:
Something seems to be broken with this one. Maybe because of the way that dialogs pass along the class attribute?

Reply:
It's weird because everything else seems to work just as before.

Reply:
Yes, because we only used the static class in `k-dialog` and now it's assigned dynamically. Changed now to `k-dialog` using whichever all.

Reply:
The upside: in Vue 3 we can get rid of the `$vnode.data.` hack as `$attrs.class` will work there as well.